src/components/pages/Login.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div class="login">
+    <p>Pour vous connecter, merci de vous créer un compte sur Github et de rentrer votre Access Token</p>
+    <form action="GET" v-on:submit.prevent="verifyUser()">
+      <label for="token">Github Access Token</label>
+      <input type="text" v-model="token">
+      <input type="submit" value="Yo">
```

Comment:
What does this component look like if there is already a token in the localStorage?